### PR TITLE
Fix vm param & useless if

### PR DESCRIPTION
**Pull request by eggggger**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License |  |
| Doc PR |  |

### CODE REVIEW

File: lib/Common.js [hunk 1]
```diff
@@ -509,11 +510,8 @@ function checkExecMode(conf) {
          ' without load balancing. To enable it remove -x option.');
   }
 
-  if (conf.instances && conf.exec_mode === undefined)
-    conf.exec_mode = 'cluster_mode';
-
   // Tell user about unstability of cluster module + Roadmap
-  if (/^cluster(_mode)?$/i.test(conf.exec_mode) &&
+  if (conf.exec_mode === 'cluster_mode' &&
```
Comment (soyuka) [inline lib/Common.js]: `cluster` and `cluster_mode` are both possible if I remember well.

Comment (eggggger) [inline lib/Common.js]: @soyuka  please see above 
494  to 500 
```
if (conf.exec_mode === 'cluster' ||
      conf.exec_mode === 'cluster_mode' ||
      conf.instances && conf.exec_mode === undefined)
    conf.exec_mode = 'cluster_mode';
  else
    conf.exec_mode = 'fork_mode';
```

Comment (soyuka) [inline lib/Common.js]: Oh indeed nice catch!

**Comment by soyuka** [comment]
Lgtm ping @Unitech

**Comment by soyuka** [comment]
ping @Unitech @vmarchaud

**Pull request closed without merging**